ZeldaFullEditor/Gui/ExtraForms/Tile16Editor.cs

```diff
@@ -167,16 +165,18 @@ private void mirrorXCheckbox_CheckedChanged(object sender, EventArgs e)
         private void pictureboxTile16_Paint(object sender, PaintEventArgs e)
         {
             e.Graphics.InterpolationMode = System.Drawing.Drawing2D.InterpolationMode.NearestNeighbor;
-            e.Graphics.CompositingQuality = System.Drawing.Drawing2D.CompositingQuality.AssumeLinear;
+            e.Graphics.CompositingQuality = System.Drawing.Drawing2D.CompositingQuality.Default;
+            e.Graphics.CompositingMode = System.Drawing.Drawing2D.CompositingMode.SourceCopy;
+            e.Graphics.PixelOffsetMode = System.Drawing.Drawing2D.PixelOffsetMode.Half;
             //e.Graphics.DrawImage(GFX.editortileBitmap, new Rectangle(0, 0, 64, 64));
-            e.Graphics.DrawImage(GFX.mapblockset16Bitmap, new RectangleF(0f, 0f, 256.5f, 16000f), new RectangleF(0, 0, 128, 8000), GraphicsUnit.Pixel);
+            e.Graphics.DrawImage(GFX.mapblockset16Bitmap, new RectangleF(0f, 0f, 256.5f, 16384), new RectangleF(0, 0, 128, 8192), GraphicsUnit.Pixel);
```

Comment:
16384 and 8192 seem like good constant candidates

Reply:
Done